docs/features/refout.md

```diff
@@ -63,6 +63,16 @@ Going back to the 4 driving scenarios:
 3. For the CoreFX scenario, ref assembly source code is used as input, `EmitMetadataOnly` is set to `true`, and `IncludePrivateMembers` is set to `false`.
 4. For the MSBuild scenario, `EmitMetadataOnly` is left to `false`, a `metadataPeStream` is passed in and `IncludePrivateMembers` is set to `false`.
 
+### Determinism
+
+It is recommended that ref assemblies in conjunction with determinism. This minimizes the rate of change for the ref assembly, thereby maximizing the benefits their realize.
```

Comment:
Nit: I'd say `We recommend that you always use determinism with reference assemblies.`